Pinta.Core/Effects/UnaryPixelOps.cs

```diff
@@ -720,19 +720,19 @@ public override ColorBgra Apply (in ColorBgra src_color)
 			color.B = Utility.ClampToByte ((intensity * 1024 + (color.B - intensity) * sat_factor) >> 10);
 
 			HsvColor hsvColor = (new RgbColor (color.R, color.G, color.B)).ToHsv ();
-			int hue = hsvColor.Hue;
+			int newHue = hsvColor.Hue;
 
-			hue += hue_delta;
+			newHue += hue_delta;
 
-			while (hue < 0) {
-				hue += 360;
+			while (newHue < 0) {
```

Comment:
The changes look fine, but just noting that this code could be improved in a future commit
These loops are effectively computing newHue mod 360, but that can be computed directly with the `%` operator, although negative numbers need special handling